include/hse/kvdb_perfc.h

```diff
@@ -95,13 +95,9 @@ enum kvdb_perfc_sidx_cnget {
     PERFC_RA_CNGET_PTOMB,
     PERFC_RA_CNGET_MULTIPLE,
 
-    /* The enumerators PERFC_LT_CNGET_GET_L0 to L5 must be sequential */
-    PERFC_LT_CNGET_GET_L0,
-    PERFC_LT_CNGET_GET_L1,
-    PERFC_LT_CNGET_GET_L2,
-    PERFC_LT_CNGET_GET_L3,
-    PERFC_LT_CNGET_GET_L4,
-    PERFC_LT_CNGET_GET_L5,
+    /* The enumerators PERFC_LT_CNGET_GET_ROOT and LEAF must be sequential */
+    PERFC_LT_CNGET_GET_ROOT,
+    PERFC_LT_CNGET_GET_LEAF,
```

Comment:
Seems like we should change the code that depends on this.  No need to use increment to next level approach since we only have root and leaves.

Reply:
Yes, I didn't look at that logic, did a blind find/replace on it.

Reply:
I can fix this in my upcoming pr if you want nabeel, would like to get your patch asap!

Reply:
Coo, thnx!